drivers/video/vnc/vnc_server.h

```diff
@@ -227,6 +228,10 @@ struct vnc_session_s
   FAR struct touch_lowerhalf_s touch; /* Touch driver instance */
 #endif
 
+#ifdef CONFIG_VNCSERVER_KBD
+  FAR struct keyboard_lowerhalf_s kbd; /* Keyboard driver instance */
```

Comment:
```suggestion
  struct keyboard_lowerhalf_s kbd; /* Keyboard driver instance */
```
and same for `struct touch_lowerhalf_s touch;` above